### ISSUE TITLE
Certain punctuation is HTML-escaped

### ISSUE THREAD

**Issue by ysb33r** [asciidoctor/asciidoctor#1362]
Given the snippet

```
[bibliography]
- [[[prag]]] Andy Hunt & Dave Thomas. The Pragmatic Programmer:
  From Journeyman to Master. Addison-Wesley. 1999.
```
The ampersand `&` arrives as `&amp;` in the converter. I don;t think asciidoctor core shoudl escape this, it is up to the specific converter to decide whether to do so.

I have also seen this behaviour in quote blocks whether a dash `-` is escaped to `&#8217`

**Comment by mojavelinux** [comment]
This is a known issue that comes from AsciiDoc's legacy bias towards SGML output. It's been previously reported as #1061.

We can't disable them right now because it would break parsing (related to #61). Therefore, it's up to the backend atm to unescape them. Both the manpage and latex backends have to do this currently. To see how the manpage backend handles it, refer to https://github.com/asciidoctor/asciidoctor/blob/master/lib/asciidoctor/converter/manpage.rb#L19

As it turns out, unescaping isn't as bad as I original thought because the entity ends up being a typed placeholder. The fact that the intermediate format is an XML entity is just an implementation detail. It's a placeholder nonetheless.

Of course, this all goes away once we have proper inline parsing.

**Comment by ysb33r** [comment]
Thanks. It turns out that for Leanpub this is not a problem, but it did catch me by surprise.

**Comment by vutunganh** [comment]
First of all, sorry for bumping this issue.

From what I understand, there is no way to make an ampersand render as `&` instead of `&amp;` in a heading. Is that right or am I missing something? Thanks.

**Comment by mojavelinux** [comment]
That's correct. The HTML converter escapes `&` as `&amp;` as it should. If you want to pass it through unmodified, you can use `pass:[&]`.
